auth/realm/base/src/main/java/org/wildfly/security/auth/realm/ElytronMessages.java

```diff
@@ -146,4 +146,10 @@
     @LogMessage(level = Logger.Level.DEBUG)
     @Message(id = 13004, value = "JAAS logout failed for principal %s")
     void debugInfoJaasLogoutFailure(Principal principal, @Cause Throwable cause);
+
+    @Message(id = 13005, value = "Filesystem-backed realm unable to decrypt identity caused by an invalid SecretKey")
```

Comment:
I think we could drop "caused by an invalid SecretKey". The `cause` will contain more details about what went wrong.

Reply:
Removed